ql/src/java/org/apache/hadoop/hive/ql/udf/generic/GenericUDAFComputeStats.java

```diff
@@ -61,6 +61,7 @@
  */
 @Description(name = "compute_stats",
       value = "_FUNC_(x) - Returns the statistical summary of a set of primitive type values.")
+@Deprecated
```

Comment:
I don't think this class should be retained - it will have no real use anymore

Reply:
Some tests use `compute_stats` directly. I have created a different JIRA https://issues.apache.org/jira/browse/HIVE-23558 to remove the UDAF and potentially rewriting/removing those tests.